libdevice/sanitizer/asan_rtl.cpp

```diff
@@ -166,7 +177,8 @@ inline uptr MemToShadow_DG2(uptr addr, uint32_t as) {
   return 0;
 }
 
-inline uptr MemToShadow_PVC(uptr addr, uint32_t as) {
+inline uptr MemToShadow_PVC(uptr addr, uint32_t as,
+                            [[maybe_unused]] const DebugInfo *debug) {
```

Comment:
Add debug info at "MemToShadow" so that we can report some types of error when mapping address is failed